src/fprime/fpp/impl.py

```diff
@@ -42,18 +42,16 @@ def fpp_generate_implementation(
     prefixes = [
         build.get_settings("framework_path", ""),
         *build.get_settings("library_locations", []),
-        build.cmake_root,
         build.build_dir / "F-Prime",
         build.build_dir,
     ]
+    if build.is_submodule_build_structure():
```

Comment:
Why?    Also, this should not use "cmake root", but build.fet_settings("project_path") should it not?

Reply:
Yes absolutely, my mistake. By using project_root, we don't need to check whether the build structure is old or new, as project_root and framework_path will simply be the same. Note, both of these values are guaranteed to exist as per the following: 
https://github.com/fprime-community/fprime-tools/blob/4fc88dc0a20fcadd5b09bb5a188bc41373237d5d/src/fprime/fbuild/settings.py#L52-L53

since [find_fprime will error out](https://github.com/fprime-community/fprime-tools/blob/4fc88dc0a20fcadd5b09bb5a188bc41373237d5d/src/fprime/fbuild/settings.py#L36) if it can't find framework_location